### PR TITLE
feat(network): adjust CNI dir

### PR DESCRIPTION
**Pull request by angao**
(cherry picked from commit 06d9906ddef5562aa19d44f35247164256805e09)

**What type of PR is this?**
/area network

**What this PR does / why we need it**:

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @hanxueluo @WangMinjie114 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: roles/container-engine/containerd/templates/config.toml.j2 [hunk 1]
```diff
@@ -23,7 +23,7 @@ disabled_plugins = ["restart"]
 
 [plugins.cri.cni]
   bin_dir = "/opt/cni/bin"
-  conf_dir = "/etc/cni/net.d"
+  conf_dir = "/etc/cni/multus/net.d"
```
Comment (hanxueluo) [inline roles/container-engine/containerd/templates/config.toml.j2:26]: 改得非常零碎。有没有什么优雅的想法？

Comment (angao) [reply roles/container-engine/containerd/templates/config.toml.j2:26]: 主要是这个目录用的地方多。。没啥好办法

Comment (angao) [reply roles/container-engine/containerd/templates/config.toml.j2:26]: 在一个 commit 里也就还好，后续改也好找

**Review by bbbmj — COMMENTED** [review]
/lgtm

**Pull request merged**